pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -269,6 +269,29 @@ func NewCmdScale(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Co
 }
 
 const (
+	autoScaleLong = `Creates an autoscaler that automatically chooses and sets the number of pods that run in a kubernetes cluster.
```

Comment:
Make a shorter first line title - move the rest to the second paragraph.  Don't use deploymentConfig, say deployment config.